mercury-framework/service-domains/customer-offer/customer-offer-common/src/main/proto/customer-offer-model.proto

```diff
@@ -155,4 +155,41 @@ message SDCustomerOffer {
       }
     }
   }
+}
+
+message retrieveCustomerOffer {
```

Comment:
It's a message, messages start with capital letters and are not verbs. Verbs are used for `rpc` calls
```suggestion
message CustomerOfferProcedure {
```